site/data/addons.yaml

```diff
@@ -42,7 +42,7 @@
   version: 3
 - id: ascanrules
   name: Active scanner rules
-  description: The release quality Active Scanner rules
```

Comment:
Have these been changed in the actual extensions? If not they will be overwritten when this data is regenerated...

Reply:
Yup that was a previous PR.